Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -99,19 +103,23 @@ public struct SupportedPlatform: Encodable, Equatable {
     ///
     /// - Since: First available in PackageDescription 5.0
     ///
-    /// - Parameter version: The minimum deployment target that the package supports.
+    /// - Parameters:
+    ///   - version: The minimum deployment target that the package supports.
     public static func macOS(_ version: SupportedPlatform.MacOSVersion) -> SupportedPlatform {
         return SupportedPlatform(platform: .macOS, version: version.version)
     }
 
     /// Configures the minimum deployment target version for the macOS platform
     /// using a version string.
     ///
-    /// The version string must be a series of two or three dot-separated integers, such as `10.10` or `10.10.1`.
-    ///
-    /// - Since: First available in PackageDescription 5.0
```

Comment:
This line probably shouldn't be deleted.

Reply:
I put them back in this file. I'll go back and see if there are more in other files.